public/src/Helpers.ts

```diff
@@ -30,18 +30,19 @@ export enum ConnStatus {
 
 const months = ["January", "February", "March", "April", "May", "June", "July", "August", "September", "October", "November", "December"]
 
-/** Returns a string with a prettier format for a deadline */
-export const getFormattedTime = (timestamp: Timestamp | undefined): string => {
+/** Returns a string with a prettier format for a timestamp */
+export const getFormattedTime = (timestamp: Timestamp | undefined, offset?: boolean): string => {
```

Comment:
When do we use the offset and when not? Should probably add a doc comment about it.